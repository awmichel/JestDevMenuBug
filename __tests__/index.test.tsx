import { renderRouter } from 'expo-router/testing-library';

// jest.mock('expo-dev-menu', () => ({
//   registerDevMenuItems: jest.fn(),
// }));

it('matches root snapshot', () => {
  const result = renderRouter();
  expect(result).toMatchSnapshot();
});
