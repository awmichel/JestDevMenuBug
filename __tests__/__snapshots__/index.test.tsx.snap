// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches root snapshot 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
